flow/include/flow/genericactors.actor.h

```diff
@@ -232,6 +232,39 @@ Future<Void> recurring(Func what, double interval, TaskPriority taskID = TaskPri
 	}
 }
 
+// Invoke actorFunc() forever in a loop
+// At least wait<interval> between two actor functor invocations
+ACTOR template <class F>
+Future<Void> recurringAsync(
+    F actorFunc, // Callback actor functor
+    double interval, // Interval between two subsequent invocations of actor functor.
+    bool absoluteIntervalDelay, // Flag guarantees "interval" delay between two subequent actor functor invocations. If
+                                // not selected, guarantees provided are "at least 'interval' delay" between two
+                                // subsequent actor functor invocations, however, due to either 'poorly choose' interval
+                                // value AND/OR actor functor taking longer than expected to return, could cause actor
+                                // functor to run with no-delay
+    double initialDelay, // Initial delay interval
+    TaskPriority taskID = TaskPriority::DefaultDelay) {
+
+	wait(delay(initialDelay));
+
+	state double beforeTS;
+	state double afterTS;
+	loop {
+		beforeTS = now();
```

Comment:
Just to make sure am on the same page, the suggestion is to obtain TS at the start of every loop, compute the `future timestamp` based on X delay param and just wait for it along with the `delayedFuture` ? 
Yeah, this sounds neat :).

Reply:
However, for the usecase where actorFunc takes unexpectedly longer (or poor choice of delay) and the desire is to have `absoluteDelay` between invocations, I see value is preserving current behavior and the current form makes code more explicit.
Thoughts?

Reply:
No what i am saying is that the following two pieces of code are equivalent in behavior:
A:
`beforeTS = now();`
`Future<Void> val = actorFunc();`
`wait(val);`
`afterTS = now();`
`double toWait = absoluteIntervalDelay ? interval : interval - (afterTS - beforeTS);`
`if (toWait > 0.0) {`
`	wait(delay(toWait));`
`}`

B:
`Future<Void> val = actorFunc();`
`Future<Void> toWait = delay(interval);`
`wait(val && toWait);`

Reply:
Because the delay is instantiated from before the wait, it effectively implicitly records the start time, and then waiting on it will wait longer if actorFunc() finishes first, but if actorFunc takes longer, it will also wait for actorFunc

Reply:
Discussed offline. All good.